2016-09-12 16:09:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец  не найден
Column  not found; SQL statement:
create table REAGENTS (
   id INT NOT NULL auto_increment,
   uuid UUID NOT NULL UNIQUE,
   name VARCHAR(200) default "",
   description CLOB default "",
   precursor BOOLEAN  default false,
   PRIMARY KEY (id)
) [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.table.Column.setDefaultExpression(Column.java:215)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3956)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5972)
	at org.h2.command.Parser.parseCreate(Parser.java:4237)
	at org.h2.command.Parser.parsePrepared(Parser.java:362)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:35:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "INSERT INTO REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, ""tes"", ""dec"" FALSE[*]) "; ожидалось "(, ., [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, ""tes"", ""dec"" FALSE[*]) "; expected "(, ., [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, null, "tes", "dec" false) [42001-192]
2016-09-12 19:35:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "tes" не найден
Column "tes" not found; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, null, "tes", "dec",  false) [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:36:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "tes" не найден
Column "tes" not found; SQL statement:
insert into REAGENTS  values(null, null, "tes", "dec",  false) [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:36:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "TEST" не найден
Column "TEST" not found; SQL statement:
insert into REAGENTS  values(null, null, `test`, "dec",  false) [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:36:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, null, [*]'tes"", ""dec"",  false)"
Syntax error in SQL statement "insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, null, [*]'tes"", ""dec"",  false)"; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, null, 'tes", "dec",  false) [42000-192]
2016-09-12 19:37:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "INSERT INTO REAGENTS (ID, ''[*]WQE, NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; ожидалось "identifier"
Syntax error in SQL statement "INSERT INTO REAGENTS (ID, ''[*]WQE, NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; expected "identifier"; SQL statement:
insert into REAGENTS (ID, ''wqe, NAME, DESCRIPTION, PRECURSOR) values(null, null, 'tes', 'dec',  false) [42001-192]
2016-09-12 19:37:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "INSERT INTO REAGENTS (ID, 'wqe'[*], NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; ожидалось "identifier"
Syntax error in SQL statement "INSERT INTO REAGENTS (ID, 'wqe'[*], NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; expected "identifier"; SQL statement:
insert into REAGENTS (ID, 'wqe', NAME, DESCRIPTION, PRECURSOR) values(null, null, 'tes', 'dec',  false) [42001-192]
2016-09-12 19:38:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "INSERT INTO REAGENTS (ID, UUID.[*]RANDOMUUID(), NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; ожидалось ",, )"
Syntax error in SQL statement "INSERT INTO REAGENTS (ID, UUID.[*]RANDOMUUID(), NAME, DESCRIPTION, PRECURSOR) VALUES(NULL, NULL, 'tes', 'dec',  FALSE) "; expected ",, )"; SQL statement:
insert into REAGENTS (ID, UUID.randomUUID(), NAME, DESCRIPTION, PRECURSOR) values(null, null, 'tes', 'dec',  false) [42001-192]
2016-09-12 19:39:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Функция "RANDOMUUID" не найдена
Function "RANDOMUUID" not found; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, UUID.randomUUID(), 'tes', 'dec',  false) [90022-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2417)
	at org.h2.command.Parser.readFunction(Parser.java:2469)
	at org.h2.command.Parser.readTermObjectDot(Parser.java:2678)
	at org.h2.command.Parser.readTerm(Parser.java:2799)
	at org.h2.command.Parser.readFactor(Parser.java:2320)
	at org.h2.command.Parser.readSum(Parser.java:2307)
	at org.h2.command.Parser.readConcat(Parser.java:2277)
	at org.h2.command.Parser.readCondition(Parser.java:2127)
	at org.h2.command.Parser.readAnd(Parser.java:2099)
	at org.h2.command.Parser.readExpression(Parser.java:2091)
	at org.h2.command.Parser.parseInsert(Parser.java:1084)
	at org.h2.command.Parser.parsePrepared(Parser.java:413)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:39:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "'qwedssf' (REAGENTS: UUID UUID NOT NULL)"
Data conversion error converting "'qwedssf' (REAGENTS: UUID UUID NOT NULL)"; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, 'qwedssf', 'tes', 'dec',  false) -- (NULL, 'qwedssf', 'tes', 'dec', FALSE) [22018-192]
2016-09-12 19:40:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Функция "UUID" не найдена
Function "UUID" not found; SQL statement:
insert into REAGENTS (ID, UUID, NAME, DESCRIPTION, PRECURSOR) values(null, uuid('2D1EBC5B7D2741979CF0E84451C5BBB1'), 'tes', 'dec',  false) [90022-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2417)
	at org.h2.command.Parser.readFunction(Parser.java:2469)
	at org.h2.command.Parser.readTerm(Parser.java:2806)
	at org.h2.command.Parser.readFactor(Parser.java:2320)
	at org.h2.command.Parser.readSum(Parser.java:2307)
	at org.h2.command.Parser.readConcat(Parser.java:2277)
	at org.h2.command.Parser.readCondition(Parser.java:2127)
	at org.h2.command.Parser.readAnd(Parser.java:2099)
	at org.h2.command.Parser.readExpression(Parser.java:2091)
	at org.h2.command.Parser.parseInsert(Parser.java:1084)
	at org.h2.command.Parser.parsePrepared(Parser.java:413)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-12 19:58:11 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.accaunt.DB_Manager.<init>(DB_Manager.java:33)
	at com.mvLab.lab.accaunt.DB_Manager.initialize(DB_Manager.java:56)
	at com.mvLab.lab.accaunt.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-12 20:41:14 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.accaunt.DB_Manager.<init>(DB_Manager.java:40)
	at com.mvLab.lab.accaunt.DB_Manager.initialize(DB_Manager.java:63)
	at com.mvLab.lab.accaunt.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-12 21:23:07 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.accaunt.DB_Manager.<init>(DB_Manager.java:40)
	at com.mvLab.lab.accaunt.DB_Manager.initialize(DB_Manager.java:63)
	at com.mvLab.lab.accaunt.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-12 21:23:40 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.accaunt.DB_Manager.<init>(DB_Manager.java:40)
	at com.mvLab.lab.accaunt.DB_Manager.initialize(DB_Manager.java:63)
	at com.mvLab.lab.accaunt.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-14 19:58:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
);"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
);"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
); [42000-192]
2016-09-14 19:58:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [42001-192]
2016-09-14 19:58:56 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [42001-192]
2016-09-14 19:59:21 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:00:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
);"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
);"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
); [42000-192]
2016-09-14 20:00:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
)"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
)"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [42000-192]
2016-09-14 20:00:30 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID)
)"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID)
)"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,

   PRIMARY KEY (rowID)
) [42000-192]
2016-09-14 20:00:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,

   PRIMARY KEY (rowID)
)"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,

   PRIMARY KEY (rowID)
)"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,

   PRIMARY KEY (rowID)
) [42000-192]
2016-09-14 20:00:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,

   PRIMARY KEY (rowID)
)"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,

   PRIMARY KEY (rowID)
)"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,

   PRIMARY KEY (rowID)
) [42000-192]
2016-09-14 20:00:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,

   PRIMARY KEY (rowID)
)"
Syntax error in SQL statement "create table [*]""REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,

   PRIMARY KEY (rowID)
)"; SQL statement:
create table "REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,

   PRIMARY KEY (rowID)
) [42000-192]
2016-09-14 20:01:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID),
   KEY(DOCUMENT, REAGENT)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [42001-192]
2016-09-14 20:01:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:01:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,


   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,


   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,


   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:01:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:01:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,

   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,

   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,

   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:03:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "("
Unknown data type: "("; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,

   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [50004-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4079)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3937)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5972)
	at org.h2.command.Parser.parseCreate(Parser.java:4237)
	at org.h2.command.Parser.parsePrepared(Parser.java:362)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-14 20:03:47 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE REAGENT_ADMISSION_TABLE_PART_ROW (
   ROWID INT NOT NULL AUTO_INCREMENT,
   ROWNUM[*] INT NOT NULL,
   DOCUMENT  INT NOT NULL,
   REAGENT INT NOT NULL,
   QUANTITY DOUBLE DEFAULT 0,
   PRICE DOUBLE DEFAULT 0,
   SUM DOUBLE DEFAULT 0,

   PRIMARY KEY (ROWID)
) "; expected "identifier"; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNum INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID)
) [42001-192]
2016-09-14 20:05:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "("
Unknown data type: "("; SQL statement:
create table REAGENT_ADMISSION_TABLE_PART_ROW (
   rowID INT NOT NULL auto_increment,
   rowNumber INT NOT NULL,
   document  INT NOT NULL,
   reagent INT NOT NULL,
   quantity DOUBLE default 0,
   price DOUBLE default 0,
   sum DOUBLE default 0,

   PRIMARY KEY (rowID),
   KEY(document, reagent)
) [50004-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4079)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3937)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5972)
	at org.h2.command.Parser.parseCreate(Parser.java:4237)
	at org.h2.command.Parser.parsePrepared(Parser.java:362)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2016-09-14 20:10:10 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:21)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:34)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-14 20:10:30 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:21)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:34)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-14 20:10:48 jdbc[5]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:187)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:129)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2016-09-15 11:21:39 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:24)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:37)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-20 18:14:53 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:24)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:37)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-20 18:29:26 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 23 more
2016-09-20 18:29:30 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 23 more
2016-09-20 18:29:31 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 23 more
2016-09-20 18:29:43 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 23 more
2016-09-21 17:32:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:32:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:38:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:39:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:39:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:39:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:41:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:41:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 17:49:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-192]
2016-09-21 22:42:50 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 23 more
2016-09-21 22:43:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, ?, ?, ?, ?) -- (NULL, ?1, ?2, ?3, ?4) [22018-192]
2016-09-21 22:50:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, ?, ?, ?, ?) -- (NULL, ?1, ?2, ?3, ?4) [22018-192]
2016-09-21 23:00:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, ?, ?, ?, ?) -- (NULL, ?1, ?2, ?3, ?4) [22018-192]
2016-09-21 23:01:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, ?, ?, ?, ?) -- (NULL, ?1, ?2, ?3, ?4) [22018-192]
2016-09-21 23:05:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Невозможно преобразование строки "2555655" в тип "DATE"
Cannot parse "DATE" constant "2555655"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, 2555655, 1, 2, 3) -- (NULL, 2555655, 1, 2, 3) [22007-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:965)
	at org.h2.table.Column.convert(Column.java:149)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 2555655
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:305)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 15 more
2016-09-21 23:05:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Невозможно преобразование строки "255" в тип "DATE"
Cannot parse "DATE" constant "255"; SQL statement:
insert into REAGENT_BALANCE (recordID, date, direction, quantity, reagent) values (null, 255, 1, 2, 3) -- (NULL, 255, 1, 2, 3) [22007-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:965)
	at org.h2.table.Column.convert(Column.java:149)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 255
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:305)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 15 more
2016-09-21 23:06:23 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:26)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:39)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-21 23:07:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, direction, quantity, reagent) values (null, ?, ?, ?) -- (NULL, ?1, ?2, ?3) [22018-192]
2016-09-21 23:10:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT NOT NULL)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT NOT NULL)"; SQL statement:
insert into REAGENT_BALANCE (recordID, direction, quantity, reagent) values (null, ?, ?, ?) -- (NULL, ?1, ?2, ?3) [22018-192]
2016-09-21 23:11:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE CREATE TABLE[*] REAGENT_BALANCE (
   RECORDID INT NOT NULL AUTO_INCREMENT,
   DIRECTION INT NOT NULL,
   DATE DATE DEFAULT NULL,
   REAGENT INT,
   QUANTITY DOUBLE DEFAULT 0,

   PRIMARY KEY (RECORDID)
) "
Syntax error in SQL statement "DROP TABLE CREATE TABLE[*] REAGENT_BALANCE (
   RECORDID INT NOT NULL AUTO_INCREMENT,
   DIRECTION INT NOT NULL,
   DATE DATE DEFAULT NULL,
   REAGENT INT,
   QUANTITY DOUBLE DEFAULT 0,

   PRIMARY KEY (RECORDID)
) "; SQL statement:
drop table create table REAGENT_BALANCE (
   recordID INT NOT NULL auto_increment,
   direction INT NOT NULL,
   date DATE default NULL,
   reagent INT,
   quantity DOUBLE default 0,

   PRIMARY KEY (recordID)
) [42000-192]
2016-09-21 23:13:18 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:38)
	at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:58)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.addConnections(PooledConnections.java:123)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:42)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections.<init>(PooledConnections.java:20)
	at org.hibernate.engine.jdbc.connections.internal.PooledConnections$Builder.build(PooledConnections.java:161)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:109)
	at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:72)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:145)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:66)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:35)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:88)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:257)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:51)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:94)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:240)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.handleTypes(MetadataBuildingProcess.java:352)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:111)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.build(MetadataBuildingProcess.java:83)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:418)
	at org.hibernate.boot.internal.MetadataBuilderImpl.build(MetadataBuilderImpl.java:87)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:691)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
	at com.mvLab.lab.account.DB_Manager.<init>(DB_Manager.java:26)
	at com.mvLab.lab.account.DB_Manager.initialize(DB_Manager.java:39)
	at com.mvLab.lab.account.Main.start(Main.java:10)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 62 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/mvLab/labAccounting/base/labBase.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 57 more
2016-09-21 23:14:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка преобразования данных при конвертации "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"
Data conversion error converting "X'aced000573720036636f6d2e6d764c61622e6c61622e6163636f756e742e636174616c6f67732e72656167656e74732e52656167656e74436174616c6f679f2e8b277e549ac90200055a0009707265637572736f724c000b6465736372697074696f6e7400124c6a6176612f6c616e672f537472696e673b4c000269647400134c6a6176612f6c616e672f496e74656765723b4c00046e616d6571007e00014c0004757569647400104c6a6176612f7574696c2f555549443b78700174000464656331737200116a6176612e6c616e672e496e746567657212e2a0a4f781873802000149000576616c7565787200106a6176612e6c616e672e4e756d62657286ac951d0b94e08b020000787000000002740004746573747372000e6a6176612e7574696c2e55554944bc9903f7986d852f0200024a000c6c65617374536967426974734a000b6d6f73745369674269747378709cf0e84451c5bbb12d1ebc5b7d274197' (REAGENT_BALANCE: REAGENT INT)"; SQL statement:
insert into REAGENT_BALANCE (recordID, direction, quantity, reagent) values (null, ?, ?, ?) -- (NULL, ?1, ?2, ?3) [22018-192]
